tasks/upgrade_debian.yml

```diff
@@ -16,6 +16,12 @@
     path: /etc/dpkg/dpkg.cfg.d/needrestart
     state: absent
 
+- name: Clean apt cache and remove unused packages
```

Comment:
Please move this task to after the `apt safe-upgrade` task because after upgrading, more packages may be unneeded, e.g. when the kernel is upgraded.

Reply:
I made this task to the last one executed. I think this is the best location because all upgrade task are so executed in a sequence and the cleanup is done after all other operations.

Reply:
I would've inserted it directly after the `apt` task as they are related. Systemd services won't change anything about cleanable or removable packages.